src/Event/CommerceAuspostEvents.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\commerce_auspost\Event;
+
+class CommerceAuspostEvents {
```

Comment:
Needs a class comment.

Could you also make this a `final` class and extend `Enum`?

Reply:
Fixed